FF1Lib/asm/0E_9030_FixPrintCharStat.asm

```diff
@@ -0,0 +1,33 @@
+char_index      = $67
+tmp             = $10 ; 16 bytes
+unsram          = $6000  ; $400 bytes
+ch_stats        = unsram + $0100  ; MUST be on page bound.  Each character allowed $40 bytes, so use 00,40,80,C0 to index ch_stats
+PrintNumber_2Digit = $8E66
+PrintNumber_3Digit = $8E70
+
+.ORG $8E04
+
+;;; we don't want to change jump targets but we need to
```

Comment:
is there that many places this is called? it would probably be best to change what calls this to call the new routine